braindecode/models/base.py

```diff
@@ -158,6 +169,15 @@ def sfreq(self):
             )
         return self._sfreq
 
+    @property
+    def add_log_softmax(self):
+        if self._add_log_softmax:
+            warnings.warn("LogSoftmax final layer will be removed! " +
```

Comment:
In this warning, can we specify that this will be changed behavior of the package and the logsoftmax won't be removed from this particular object when accessing the property?